.vscode/launch.json

```diff
@@ -39,6 +39,17 @@
       "justMyCode": true,
       "cwd": "${workspaceFolder}"
     },
+    {
```

Comment:
this config and environment path seems to be specific to your machine

Reply:
Fixed, thanks